### PR TITLE
Feature/admin help page

### LINKED ISSUE #2241: Local documentation browser
Now that we ship the documentation in HTML in the admin, we should provide a simple menu that allows to open any of the guides that are on the server

Tagging for 7.0 but could be postponed

### PR DESCRIPTION
**Pull request by cgx**
# Description
New help page in the Web admin interface where to find all links to the guides, the installed version number and the server's hostname. We could add more information; suggestions are welcomed.

# Issue
Fixes #2241 

# Delete branch after merge
YES

# NEWS file entries
## New Features
* Web admin: new help page with links to all the guides

# Preview
pf-admin-help

### CODE REVIEW

File: html/pfappserver/root/admin/help.tt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          guide('Checkpoint firewall', 'Checkpoint_Quick_Install_Guide') | none;
+          guide('CoovaChilli access controller', 'CoovaChilli_Quick_Integration_Guide') | none;
+          guide('FortiGate firewall', 'FortiGate_Quick_Install_Guide') | none;
+          guide('Microsoft PKI (MSPKI)', 'MSPKI_Quick_Install_Guide') | none;
+          guide('MobileIron mobile device manager', 'MobileIron_Quick_Install_Guide') | none;
+          guide('OpenWrt', 'OpenWrt-Hostapd-15-05_Quick_Install_Guide') | none;
+          guide('OPSWAT Metadefender Endpoint Management', 'OPSWAT_Quick_Install_Guide') | none;
+          guide('PaloAlto firewall', 'PaloAlto_Quick_Install_Guide') | none;
+          guide('Symantec Endpoint Protection Manager', 'SEPM_Quick_Install_Guide') | none;
+          %]
```
Comment (julsemaan) [inline html/pfappserver/root/admin/help.tt]: Seems to be missing the SentinelOne quick install guide

Comment (julsemaan) [reply html/pfappserver/root/admin/help.tt]: Also might be good considering to auto-populate this list from the files in the doc directory

Comment (cgx) [reply html/pfappserver/root/admin/help.tt]: The SentinelOne Guide is also missing on the website .. 👎

Comment (extrafu) [reply html/pfappserver/root/admin/help.tt]: Normal for now, it was added for v7.

**Pull request merged**